pyabc/visualization/sample.py

```diff
@@ -312,10 +317,18 @@ def plot_acceptance_rates_trajectory(
         # note: the first entry of time -1 is trivial and is thus ignored here
         h_info = history.get_all_populations()
         times.append(np.array(h_info['t'])[1:])
-        samples.append(np.array(h_info['samples'])[1:])
         pop_sizes.append(np.array(
             history.get_nr_particles_per_population().values[1:]))
 
+        if normalize_by_ess:
```

Comment:
I think there is one error here: Not samples should be replaced by ESS, but pop_sizes should be replaced by ESS, t.t. the resulting rations are usually <= 1. So, just move this code block up a bit ;) Sorry, if I miscommunicated that.

Reply:
Please also update the docstring accordingly.

Reply:
done!